numpy/core/arrayprint.py

```diff
@@ -1,3 +1,17 @@
+import contextlib
+import warnings
+from .overrides import array_function_dispatch, set_module
+from .numerictypes import (longlong, intc, int_, float_, complex_, bool_,
+                           flexible)
+from .numeric import concatenate, asarray, errstate
+from .fromnumeric import ravel, any
+from .multiarray import (array, dragon4_positional, dragon4_scientific,
+                         datetime_as_string, datetime_data, ndarray,
+                         set_legacy_print_mode)
+from . import multiarray
+from .umath import absolute, not_equal, isnan, isinf, isfinite, isnat
+from . import numerictypes as _nt
+import numpy as np
```

Comment:
Move these lines back to where they were.

Reply:
Done. Looks like my text editor Atom made these changes when I saved and I didn't check.